src/main/java/seedu/address/logic/commands/AddInterviewCommand.java

```diff
@@ -114,8 +114,8 @@ public CommandResult execute(Model model) throws CommandException {
         }
         model.addInterview(interview);
         model.sortInterview();
-        applicantSearch.updateCurrentStatusToReflectInterview(model);
-        interviewerSearch.updateCurrentStatusToReflectInterview(model, applicantSearch);
+        applicantSearch.updateCurrentStatusToReflectScheduledInterview(model);
```

Comment:
Good catch. In the future I will remember to update the team when I change method names so others may follow suit quickly and avoid compile errors